docs/content/docs/grype/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Tech Preview - Grype Integration"
+linkTitle: "Access"
+weight: 1
+---
+
+Anchore Engine 0.10 includes a limited-functionality tech preview of an integration with [Grype](https://github.com/anchore/grype)
+for vulnerability scanning. This integration will replace the legacy vulnerability scanner in a future version of Anchore Engine,
+but is provided in this release in a preview capacity so users can try it out.
+
+***Note:*** This tech preview is not intended for use in production environments. It should be installed in sandbox exnvironments,
+and is strictly provided here so users can get start to use it and get a sense for the functionality. It may not include all of the
+functionality of the legacy scanner. Please report any issues found with it on the
+[anchore-engine Github repo](https://github.com/anchore/anchore-engine/issues).
+
+### Installing
+As of 0.10.0, Anchore Engine can be configured to use either the legacy or grype vulnerability scanner. It is not possible to run
+both vulnerability scanners at the same time. This configuration is picked up at bootstrap, and cannot be changed ona running system. 
+
+#### Running with docker-compose
+1. Install or update to Anchore Engine 0.10.0.
```

Comment:
I'd like to get confirmation that these configuration options are correct, particularly for helm as I've not run it that way yet. (I based this off of what I saw in your charts PR, @Btodhunter

Reply:
this is right. the other variable is ANCHORE_GRYPE_DB_URL, but that only needs to be configured differently for enterprise